### PR TITLE
inline out variables

### PR DESCRIPTION
**Pull request by SimonCropp**
and update editor config with the setting

### CODE REVIEW

File: src/NuGetGallery.Core/Infrastructure/TableErrorLog.cs [hunk 1]
```diff
@@ -61,8 +61,7 @@ void ITableEntity.ReadEntity(IDictionary<string, EntityProperty> properties, Mic
         {
             // This can occasionally fail because someone didn't finish creating the entity yet.
 
-            EntityProperty value;
-            if (properties.TryGetValue("SerializedError", out value))
+            if (properties.TryGetValue("SerializedError", out var value))
```
Comment (erdembayar) [inline src/NuGetGallery.Core/Infrastructure/TableErrorLog.cs:64]: Is it possible to make suggestion like below which is more readable? Not sure if there's rule for it in editorconfig?
       `if (properties.TryGetValue("SerializedError", out EntityProperty value))`

Comment (SimonCropp) [reply src/NuGetGallery.Core/Infrastructure/TableErrorLog.cs:64]: given you already have 163 instance of ` out var` in the codebase, i dont think this is a valid reason to block this PR

Comment (SimonCropp) [reply src/NuGetGallery.Core/Infrastructure/TableErrorLog.cs:64]: IMO a more readable version is 
```
if (properties.TryGetValue("SerializedError", out var serializedError))
```

Comment (erdembayar) [reply src/NuGetGallery.Core/Infrastructure/TableErrorLog.cs:64]: We haven't had time to update our coding guidelines, I think #10 from https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md is very good guidance.

> given you already have 163 instance of  out var in the codebase, i dont think this is a valid reason to block this PR

That is the historic reason, if you look at most recent PRs, we're trying to use more of explicit types.

**Comment by SimonCropp** [comment]
i will resubmit this once https://github.com/NuGet/NuGetGallery/pull/10046 is merged

**Pull request closed without merging**